### PR TITLE
add curlbash to linux install options + cleanup faq

### PR DESCRIPTION
**Pull request by brewn**
Signed-off-by: brewn <nbrewer@chef.io>

Not sure if we want to list the curlbash method as an option for macos as well, or if we'd prefer to favor package manager options where they're available.

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: www/source/partials/docs/_using-hab-install-faq.html.md.erb [hunk 1]
```diff
@@ -17,14 +17,7 @@ A: We've got you covered! The script we provide for doing curlbash installations
...
 
-Install Habitat via curlbash using the following command:
-`curl https://raw.githubusercontent.com/habitat-sh/habitat/master/components/hab/install.sh | sudo bash`
-
-If you're staunchly in the anti-curlbash camp you can get the latest packages from our [download page](https://habitat.sh/downloads) or via releases on [our bintray page](https://bintray.com/habitat)
-
-**Q: Er, how do I install habitat?**
-
-A: You'll need to download the `hab` package. Extract the binary to somewhere on your $PATH (or somewhere that you can execute the binary from a command line). If you're installing on a workstation we would strongly recommend running `hab setup` after youve extracted the binary. Read our [step by step docs here](/docs/using-habitat/#install-habitat).
+If you're staunchly in the anti-curlbash camp, you can get the latest packages from the [download links](/docs/install-habitat/#install-habitat) listed previously, or via releases on our [bintray page](https://bintray.com/habitat)
```
Comment (cnunciato) [inline www/source/partials/docs/_using-hab-install-faq.html.md.erb]: Little thing, but missing a period. And should Bintray be capped?

File: www/source/partials/global/_download-and-install.slim [hunk 2]
```diff
@@ -9,6 +9,10 @@ hr
       blockquote Requires 64-bit processor with kernel 2.6.32 or later
       p Once you have downloaded the package, extract the hab binary with tar to <code>/usr/local/bin</code> or add its location to your <code>PATH</code> (e.g. <code>tar -xvzf hab.tgz -C /usr/local/bin --strip-components 1</code>).
       = link_to 'Download Habitat for Linux', 'https://api.bintray.com/content/habitat/stable/linux/x86_64/hab-%24latest-x86_64-linux.tar.gz?bt_package=hab-x86_64-linux', class: 'button cta'
+      p Alternatively, you can install Habitat via the command line by downloading and running the installation script:
+      = code(:shell) do
+          |
+           curl https://raw.githubusercontent.com/habitat-sh/habitat/master/components/hab/install.sh | sudo bash
```
Comment (cnunciato) [inline www/source/partials/global/_download-and-install.slim]: :awesome:

Comment (cnunciato) [reply www/source/partials/global/_download-and-install.slim]: This saves me from either `curl|bash`-spamming #general or searching Slack for it. :)

File: www/source/partials/docs/_using-hab-install-faq.html.md.erb (hunk 1 above)
Comment (brewn) [reply www/source/partials/docs/_using-hab-install-faq.html.md.erb]: That actually brings up a good point about how Habitat itself is kind of inconsistently capitalized in this doc - older stuff seems to stick with lowercase while more recently we've been capitalizing it.

Comment (cnunciato) [reply www/source/partials/docs/_using-hab-install-faq.html.md.erb]: Ah, yeah. I'm basically capitalizing that everywhere, unless it's a reference to the actual `hab` binary of course.

File: www/source/partials/docs/_using-hab-install-faq.html.md.erb [hunk 3]
```diff
@@ -11,21 +11,14 @@ A: We publish compiled packages for OSX, Linux, and Windows. `hab` has a require
...
 
 **Q: How do I install `hab` across my server fleet?**
 
-A: for the most part, we leave that up to you. You could just use the aforementioned curl-bash with your provisioner of choice. If your app was dockerized with habitat then you won't even need to ask this question, because you'll have everything you need inside your container. We are working on first class mesos dc/os, cloudfoundry and kubernetes integrations - which you can keep up to date on in [our best practices section](/docs/best-practices/) and [blog](/blog).
+A: for the most part, we leave that up to you. You could just use the aforementioned curl-bash with your provisioner of choice. If your app was dockerized with Habitat then you won't even need to ask this question, because you'll have everything you need inside your container. We are working on first class Mesosphere DC/OS, Cloud Foundry, and Kubernetes integrations - which you can keep up to date on in [our best practices section](/docs/best-practices/) and [blog](/blog).
```
Comment (cnunciato) [inline www/source/partials/docs/_using-hab-install-faq.html.md.erb]: `For the most...` (cap F)

Sorry if I'm killing you on these. :)

Comment (brewn) [reply www/source/partials/docs/_using-hab-install-faq.html.md.erb]: You're a great docs person 👍

**Comment by cnunciato** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/354471132).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**